### PR TITLE
feat: block explorer module, command improvements

### PR DESCRIPTION
**Pull request by JackHamer09**
# What :computer: 
* Added Block Explorer module as an option
* Improved info logs
* Added support for stopping/restarting/cleaning only single module

# Why :hand:
* Block Explorer module added for an easy local setup
* Make information logs easier to read
* In case you want to clean/stop/restart a single module, like Block Explorer, you can now do that with `zksync-cli dev stop [module-name]`

# Evidence :camera:
image

# Notes 📝
- Block Explorer module source code: https://github.com/matter-labs/zkcli-block-explorer

### CODE REVIEW

**Review by dutterbutter — COMMENTED** [review]
I hit this issue during setup:

```
Please make sure you have the correct access rights
and the repository exists.
ⓘ There was an error while starting the testing environment:
ⓘ Command exited with code 128:
```

Steps ran:
1. Install deps
2. Build 
3. `./bin/index.js dev config`  - Choose local-setup + BE
4. `./bin/index.js dev start`

File: src/utils/git.ts [hunk 1]
```diff
@@ -26,3 +26,23 @@ export const cloneRepo = async (repoUrl: string, destination: string) => {
   Logger.debug(`Cloning ${repoUrl} repository to ${destination}`);
   await executeCommand(command);
 };
+
+export const getLatestReleaseVersion = async (repo: string): Promise<string> => {
```
Comment (dutterbutter) [inline src/utils/git.ts:30]: Is this called anywhere?

File: src/utils/files.ts [hunk 2]
```diff
@@ -34,3 +34,10 @@ export const writeFile = (filePath: string, data: string | NodeJS.ArrayBufferVie
   // Then write file
   fs.writeFileSync(filePath, data, "utf-8");
 };
+
+export const createSymlink = (targetPath: string, linkPath: string, type: "file" | "dir" | "junction" = "file") => {
```
Comment (dutterbutter) [inline src/utils/files.ts:38]: Is this called anywhere?

File: src/utils/git.ts (hunk 1 above)
Comment (JackHamer09) [reply src/utils/git.ts:30]: These are utils that modules can use

File: src/utils/files.ts (hunk 2 above)
Comment (JackHamer09) [reply src/utils/files.ts:38]: Same. These are utils that modules use. Like common functionality that modules usually require.

**Review by dutterbutter — APPROVED** [review]

**Pull request merged**